src/System/Wlog/Severity.hs

```diff
@@ -44,6 +47,10 @@ data LogRecord = LR !Severity !Text deriving Show
 allSeverities :: Set Severity
 allSeverities = Set.fromAscList [minBound .. maxBound]
 
+-- | Empty 'Set' of 'Severity'.
+emptySeverities :: Severities
+emptySeverities = Set.empty
```

Comment:
I think we don't need `emptySeverities` function because it's just `mempty` (`Set` is a `Monoid`).

Reply:
@ChShersh Oh, right! No need in useless functions.